packages/lexical-website/tsconfig.json

```
@@ -0,0 +1,7 @@
+{
+  "extends": "@tsconfig/docusaurus/tsconfig.json",
```

I don't know if this has any specific compiler options that are required by docusaurus, but you can always just extends our base config:

```suggestion
  "extends": "../../tsconfig.json",
```

---

We do need these docusaurus compile options for typescript to know the types and the`@site` imports, I couldn't extend from two tsconfigs so hence why this is happening, I could however add the docusaurus typescript options to the base tsconfig but I don't know how you guys feel about that one. I could also copy paste the base config options into the website tsconfig.